editor/modes/visual-editor/block.js

```diff
@@ -133,6 +134,17 @@ class VisualEditorBlock extends wp.element.Component {
 
 		const { onSelect, onStartTyping, onHover, onMouseLeave, onFocus, onInsertAfter } = this.props;
 
+		// Normalize controls as an array of arrays (toolbars of controls)
```

Comment:
Yep, but this component will have the following props : content, attributes, setAttributes. Or we could pass a block uid instead and duplicate `setAttributes`. Having `attributes` and `setAttributes` as props for a `GroupedToolbars` might not be a good idea.